### PR TITLE
Authentication conversion

### PR DESCRIPTION
**Pull request by thongsav-usgs**

### CODE REVIEW

File: coastal-hazards-portal/pom.xml [hunk 1]
```diff
@@ -128,6 +139,10 @@
 		    <artifactId>jersey-mvc-jsp</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>org.glassfish.jersey.media</groupId>
+			<artifactId>jersey-media-moxy</artifactId>
```
Comment (jiwalker-usgs) [inline coastal-hazards-portal/pom.xml]: What does moxy add?

**Comment by jiwalker-usgs** [comment]
How does CIDA_AUTHORIZED_ROLE get set?

File: coastal-hazards-portal/pom.xml (hunk 1 above)
Comment (thongsav-usgs) [inline coastal-hazards-portal/pom.xml]: it's annoying, but the auth client needs it to deserialize the token coming back from the auth webservice. I believe JAXB would also work if we ever needed to switch.

**Comment by thongsav-usgs** [comment]
It's arbitrarily set by the app, we will switch to using a real role, likely "CCH_ADMIN".

**Pull request merged**